stock_orderpoint_uom/models/procurement_order.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class ProcurementOrder(models.Model):
+    _inherit = "procurement.order"
+
+    @api.multi
+    @api.onchange('product_id')
+    def onchange_product_id(self):
```

Comment:
I'm getting an error in runbot when creating a new procurement order and selecting a product, in the on_change method.